frontend/src/components/calendar_frontend/calendar.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import CalendarDays from './calendar-days';
+import './calendar.css';
+
+export default function Calendar() {
+  const weekdays = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+  const [currentDay, setCurrentDay] = useState(new Date());
+
+  const changeCurrentDay = (day) => {
+    setCurrentDay(new Date(day.year, day.month, day.number));
+  }
+
+  const nextDay = () => {
+    setCurrentDay(new Date(currentDay.setDate(currentDay.getDate() + 1)));
+  }
+
+  const previousDay = () => {
+    setCurrentDay(new Date(currentDay.setDate(currentDay.getDate() - 1)));
+  }
+
+  return (
+    <div className="calendar">
+      <div className="calendar-header">
+        <div className="title">
+          <h2>{months[currentDay.getMonth()]} {currentDay.getFullYear()}</h2>
+        </div>
+        <div className="tools">
+          <button onClick={previousDay}>
+            <span className="material-icons">
```

Comment:
you forgot add CSS for this span tag? Is that the reason why your arrow doesn't look like arrow?